### PR TITLE
[wip]Family heirloom edits(give suggestions)

### PR DESCRIPTION
**Pull request by improvedname**
Scope of the PR is to give every job something special. what they could gotten from their family. like a special clip that holds more then others. or a machete that gone trough down families and such.

also turning family heirlooms from negative to positive. since it always been a positive thing.

- [ ] add a special item for every job

## Changelog (neccesary)
:cl: Improvedname [wip changelog]
tweak: Changes Family Heirloom quirk from being a negative one. to an positive one.
/:cl:

### CODE REVIEW

**Comment by pireamaineach** [comment]
Y'change-log's borked.

**Comment by improvedname** [comment]
im not setting it untill i finish it silli

**Comment by ScratKnapp** [comment]
Heirloom binoculars for Rangers/Explorers
Flasks for vaulters
Extra-capacity cell for BoS
Coins for wasters maybe??

File: code/datums/traits/good.dm [hunk 1]
```diff
@@ -256,3 +256,76 @@
...
+			heirloom_type = /obj/item/book/manual/wiki/security_space_law
+		if("NCR Trooper", "NCR Recruit", "NCR Heavy Trooper")
+			heirloom_type = pick(/obj/item/kitchen/knife/combat,
+			/obj/item/kitchen/knife/combat/survival,
+			/obj/item/toy/cards/deck,
+			/obj/item/lighter)
+		if("NCR Lieutenant")
+			heirloom_type = /obj/item/ammo_box/a762/qaudstacked
+		if("Legion Explorer")
+			heirloom_type = /obj/item/ammo_box/a762/qaudstacked
```
Comment (Light-Weave) [inline code/datums/traits/good.dm]: quad*

**Comment by ChronicPwnage** [comment]
I think consumable items (ie: ammo) being an heirloom is dumb. Who treasures an unused strip of ammo as their family heirloom?

**Comment by pireamaineach** [comment]
It's not the ammo, it's the clip itself.

**Comment by ChronicPwnage** [comment]
> 
> 
> It's not the ammo, it's the clip itself.

That's not my point, I get that you could use the ammo and then keep carrying around the empty clip, but that just further accentuates the ridiculous nature of having a clip of ammunition as a family heirloom.

**Comment by pireamaineach** [comment]
That's not that ridiculous actually. Pretty sure I met some yank that revered his grandfather's old M1 Garand clip.

**Comment by ChronicPwnage** [comment]
> 
> 
> That's not that ridiculous actually. Pretty sure I met some yank that revered his grandfather's old M1 Garand clip.

I personally think that is dumb, and would prefer a watch or even a gun or something. It seems really odd to revere something so disposable, it's like people that buy celebrity napkins for hundreds of dollars. Weird as fuck.

**Comment by pireamaineach** [comment]
Okay. Doesn't change that people have done it. It's not an unreasonable item to get as a heirloom.

**Comment by GhostEcho** [comment]
If you want a gun heirloom, it's going to be something like a zipgun

**Comment by pireamaineach** [comment]
Raider and wastelander zipgun heirlooms wouldn't be half bad.

**Comment by improvedname** [comment]
The thong about the clip is. They atleast reload your starting gun twice. But on the cost you can't lose it or you get depressed. So you effeciently turn 2 clips into 1 if you get this for a extra slot on your belt/pockets. As you can fully load this clip up with 2 doublepackers  so it really isn't disposable. See it an r81 magazine. 20 bullets. Nobody would ditch it since it holds alot of ammo. And is a limited thing. This speciall clip is a limited thing. From your family so you have to keep it. But in trade it grants you 10 more capicity effeciently turning it into a 81 magazine but for coltmasters. Its not about straight play. Yes sure they are disposable but so are 81 magazines etc. But you want to keep special magazines around and on you since they are worth to refill. Unless yknow you only like to get dice lighter or a deck of cards

**Comment by improvedname** [comment]
Again. You don't have to pick this trait. Its optional its not forced.

**Comment by improvedname** [comment]
For the wastelanders its gonna be stuff like clips and such. Since theres nothing special since after the nuke that survived. For the vault dwellers its gonna be stuff from watches etc. Whatever you would grab before getting nuked

**Comment by ChronicPwnage** [comment]
Wait wait wait, it's a 20 round clip? For a trait that GIVES you points? I think that's kinda OP actually, why would that make sense? Heirloom is basically a free point right now anyways, it's very fucking easy not to lose something as if someone gets into your backpack chances are you're dead anyways. So why would we make it EVEN STRONGER?

That's why I want to go more towards stuff you'd have an actual chance at losing like a pistol or a pair of binoculars, something you'd hold in your hand that you might drop.

**Comment by improvedname** [comment]
You missed the point where it costs a point now

**Comment by pireamaineach** [comment]
Heirloom zip gun for raiders.

**Comment by RenGusta** [comment]
Month old at this point, Reopen if you ever come back to it.

**Pull request closed without merging**